include/odp/api/spec/packet_io.h

```diff
@@ -568,6 +583,15 @@ typedef struct odp_pktio_capability_t {
 
 	/** @deprecated Use enable_loop inside odp_pktin_config_t */
 	odp_bool_t ODP_DEPRECATE(loop_supported);
+
+	/** Maximum timeout in nanosec to wait for producer form the vector of packets
+	 * event(odp_packet_vect_t).
+	 * The value zero denotes the producer is not capablable of producing
+	 * the vector packet events a.k.a odp_packet_vect_t.
+	 * @see odp_packet_vect_t
+	 * @see odp_pktin_vector_prod_config_t::max_vector_prod_tmo_ns
+	 */
+	uint64_t max_vector_prod_tmo_ns;
```

Comment:
Maybe a struct is needed instead of using tmo capa for supported/not supported. Max vector size in packet input could differ from max vector size in other parts of the system, so size is likely needed here as well.

struct {
    odp_bool_t supported;
    uint32_t max_size;
    uint64_t max_tmo_ns;
} pktin_vector;

Reply:
Ack

Reply:
`uint32_t min_size` could be also useful.